charts/k8ssandra/values.yaml

```diff
@@ -482,8 +500,11 @@ medusa:
   # -- Enables usage of a bucket across multiple clusters.
   multiTenant: false
 
-  # -- API interface used by the object store. Supported values include `s3`, 's3_compatible'
-  # and `gcs`. For pod mapped storage, use 'local'
+  # -- API interface used by the object store. Supported values include `s3`,
+  # 's3_compatible' and `gcs`. For file system storage, i.e., a pod volume
+  # mount, use 'local'. Note that 'local' does not necessarily imply a local
+  # volume. It could also be network attached storage. It is simply accessed
+  # through the local file system.
```

Comment:
Or accessed through pod's mounts? "local file system" is a bit odd here as you just said it's not local necessarily.

Reply:
I will remove local.